io.github.kaschpal.timetable.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "io.github.kaschpal.timetable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "timetable",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=xdg-documents",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
you shouldn't need the dconf permissions when using the GNOME 3.36 runtime, please remove.